CONTRIBUTING.md

```diff
@@ -93,13 +93,25 @@ Alternatively, you can run the tests inside of vscode. There are several vscode
 
 1. Double-check the `CHANGELOG.md` contains all desired change comments
    and has the version to be released with date at the top.
-1. Double-check that the extension `package.json` has the version you intend to release.
-   If you are doing a patch release (as opposed to minor or major version) this should already
-   be correct.
-1. Trigger a release build on Actions by adding a new tag on branch `main` of the format `vxx.xx.xx`
+1. Double-check that the extension `package.json` has the version you intend to release. If you are doing a patch release (as opposed to minor or major version) this should already be correct.
+1. Create a PR for this release:
+    * This PR will contain any missing bits from steps 1 and 2. Most of the time, this will just be updating `CHANGELOG.md` with today's date.
+    * Create a new branch for the release named after the new version. For example: `v1.3.6`
+    * Create a new commit with a message the same as the branch name.
+    * Create a PR for this branch.
+    * Wait for the PR to be merged into `main`
+1. Trigger a release build on Actions by adding a new tag on branch `main` named after the release, as above. Note that when you push to upstream, you will need to fully qualify the ref. A command like this will work:
```

Comment:
Future: would it be useful to have a workflow dispatch event for this too? Then you can release from the UI.

Reply:
Possibly, though by doing it locally, you are (I am) more likely to go over everything in detail.